sideboard/config.py

```diff
@@ -38,36 +99,64 @@ def get_dirnames(pyname, plugin):
     package under the configured "plugins_dir" directory.
 
     These two directories are also automatically inserted into plugin config
-    files as "root" and "module_root" and are available for interpolation.  For
-    example, a plugin could have a line in their config file like
+    files as "root" and "module_root" and are available for interpolation. For
+    example, a plugin could have a line in their config file like::
+
         template_dir = "%(module_root)s/templates"
+
+
     and that would be interpolated to the correct absolute path.
 
-    This function takes two parameters:
-    pyname: the __file__ of the module needing to know these directories
-    plugin: boolean indicating whether this is a plugin or Sideboard itself
+    Parameters:
+        requesting_file_path (str): The __file__ of the module requesting the
+            "root" and "module_root" directories.
+        is_plugin (bool): Indicates whether a plugin is making the request or
+            Sideboard itself is making the request.
     """
-    module_dir = os.path.dirname(os.path.abspath(pyname))
-    if plugin:
+    module_dir = os.path.dirname(os.path.abspath(requesting_file_path))
+    if is_plugin:
         from sideboard.lib import config
         plugin_name = os.path.basename(module_dir)
-        root_dir = os.path.join(config['plugins_dir'], plugin_name.replace('_', '-'))
+        root_dir = os.path.join(config['plugins_dir'], plugin_name)
```

Comment:
What was the reason for getting rid of the `replace` here? That seems like something that could break some plugins if they were expecting underscores to be replaced with hyphens.